R/generate-docs.R

```diff
@@ -3,29 +3,57 @@
 #' This function is the main entry point for creating validation docs.
 #' @param specs tibble of requirements in the format returned by
 #'   [read_spec_gsheets()].
-#' @param test_output_dir path to direction containing a test output files. See
-#'   [input_formats].
+#' @param auto_test_dir,man_test_dir path to directories containing automatic
+#'   and manual test output files. See [input_formats].
 #' @param output_dir Directory to write the output documents to. Defaults to
 #'   working directory.
 #' @importFrom dplyr full_join rename
 #' @importFrom tidyr unnest
 #' @export
 create_validation_docs <- function
 (
-  requirements, test_output_dir, output_dir = getwd()
+  requirements, auto_test_dir = NULL, man_test_dir = NULL,
```

Comment:
Not changed in this PR, but there's a code/documentation mismatch for the first parameter.  (The documentation was updated but not the code when "requirements" was extended to handle stories.) I'll push that fix to this PR since the line's being touched.